lib/connection/connect.js

```diff
@@ -236,6 +236,9 @@ function makeConnection(family, options, callback) {
       socket = tls.connect(parseSslOptions(family, options));
     } else {
       socket = net.createConnection(parseConnectOptions(family, options));
+      if (typeof socket.disableRenegotiation === 'function') {
```

Comment:
aha, related to our previous discussion, you are calling this on the _non_ TLS socket :) It might make sense to move this outside of the `useSsl` block, but I think it only applies to tls sockets.